pages/instructorAssessmentQuestionManualGrading/instructorAssessmentQuestionManualGrading.js

```diff
@@ -0,0 +1,98 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const { error, sqlDb, sqlLoader} = require('@prairielearn/prairielib');
+
+const ltiOutcomes = require('../../lib/ltiOutcomes');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+function isGradable(instanceQuestion) {
+    return !instanceQuestion.graded_at;
+}
+
+router.get('/', function(req, res, next) {
+    debug('GET /');
+    var params = {
+        assessment_id: res.locals.assessment.id,
+        assessment_question_id: res.locals.assessment_question_id,
+    };
+    sqlDb.queryOneRow(sql.select_question, params, (err, result) => {
+        if (ERR(err, next)) return;
+        
+        // Instance question doesn't exist
+        if (result.rowCount == 0) {
+            return error.make(404, 'Assessment question not found.');
+        }
+
```

Comment:
This check won't do anything because we are already requiring exactly one row to be returned by calling `queryOneRow`.
```suggestion
```